smalt/0.7.6/Dockerfile

```diff
@@ -18,21 +18,22 @@ LABEL maintainer.email1="jvhagey@gmail.com"
 # Install dependencies
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update &&\
-    apt-get install -y \
-        autoconf \
-        pkg-config \
-        gcc \
-        libtool \
-        libcurl4-openssl-dev \
-        libncurses5-dev \
-        zlib1g-dev \
-        libbz2-dev \
-        liblzma-dev \
-        python \
-        make \
-        gawk \
-        unzip \
-        wget && \
+    apt-get install -y --no-install-recommends \
```

Comment:
You added versions to all your installations!

This might be overkill.

There are default versions for every tool in ubuntu distributions (They can be looked up at https://packages.ubuntu.com/)

Reply:
I kinda thought it was overkill, but https://hadolint.github.io/hadolint/ said to an as that was in the check list I decided to do it. Want me to change it back then?

Reply:
Maybe I should change what linter I'm sending people to...